packages/ui/package.json

```
@@ -14,14 +14,18 @@
     "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0"
   },
   "peerDependencies": {
+    "@radix-ui/react-avatar": "^1.1.0",
     "@uzh-bf/design-system": "2.13.3",
```

```suggestion
    "@uzh-bf/design-system": "^2.13.3",
```

peer dependencies should have a ^ to allow some flexibility. all other deps we pin, beside @types deps which also have a ^